File: exercicios/1165/1165.py
n = int(input())

for i in range(0,n):
    v = int(input())
    s = 0
    j = 1
    while j <= v:
        if v % j == 0:
            s = s + 1
        j +=1
    if s > 2:
        print('{} nao eh primo'.format(v))
    else:
        print('{} eh primo'.format(v))

File: exercicios/R/1165/1165.r
input = file('stdin','r')

n = as.integer(readLines(input,n=1))

for(i in 1:n){
    s = 0
    j = 1
    num = as.integer(readLines(input,n=1))

    while(j<=num){
        if(num%%j==0){
            s = s + 1
        }
        j = j + 1
    }
    if(s > 2){
        write(sprintf("%d nao eh primo",num),'')
    }else {
       write(sprintf("%d eh primo",num),'')
    }
}